doc/quickstart/geoserver_quickstart.rst

```diff
@@ -29,35 +29,32 @@ This Quick Start describes how to:
   * add a layer from a NetCDF file
 
 .. contents:: Contents
+   :local:
   
-Start |GS|
-================================================================================
+Start Geoserver
+===============
 
-#. Select the |osgeolive-appmenupath-geoserver| in the menu.
```

Comment:
Replacement for |osgeolive-appmenupath-geoserver| as :menuselection:`Geospatial --> Web Services --> GeoServer --> Start GeoServer` has been introduced while shell-scripts can change installation path so we do not have to update docs.

Reply:
Okay, I will revert.